components/raftstore/src/coprocessor/config.rs

```diff
@@ -124,6 +128,16 @@ impl Config {
         if self.region_bucket_size.0 == 0 {
             return Err(box_err!("region_bucket size cannot be 0."));
         }
+        if self.region_bucket_merge_size.0 == 0 {
+            return Err(box_err!("region-bucket-merge-size cannot be 0."));
+        }
+        if self.region_bucket_merge_size.0 >= self.region_bucket_size.0 {
```

Comment:
Good point.  It's possible that two buckets are merged and then in next round it's split again if the new merged bucket has some write traffic and leads to the actual size grows to bigger than configured region_bucket, which is legitimate. 

But if the config of region_bucket_merge_size is more than half of configured region_bucket_size (user changes the default value), then merged bucket size could be larger than region_bucket_size which could be split in next round. This is not legitimate. 
So to prevent this happening, the check here should be if self.region_bucket_merge_size.0 >= self.region_bucket_size.0/2.